README.md

```diff
@@ -5,19 +5,66 @@ Stores all the manifests and corresponding image-infos to test OSbuild.
 
 ### Update workflow
 
-The update workflow starts with a github action that is manually triggered. This
-GH action is named `Update DB`. It will firstly clone the latest
-`osbuild-composer` and copy all its manifests into the database. Secondly the
-action will create a commit and push this on the corresponding gitlab ci
-project
-(https://gitlab.com/redhat/services/products/image-builder/ci/manifest-db).
-Thirdly, after the push is completed, the action triggers a special pipeline on
-the CI. This pipeline builds every manifest from the DB. Once building is done,
-the pipeline starts the `schutzbot/include_image_info.sh` script. This script
-will download all the produced image-infos resulting from the builds and update
-the DB with them. Then it will amend the commit sent to the CI pipeline with
-these freshly built image-info, push this commit to github on a branch and
-create a pull request out of it.
+The update workflow is made of 3 distinct steps:
+
+#### First step, triggering the CI
+
+The update workflow starts with a `Update DB` GitHub action that is manually
+triggered. This action imports a subset of the manifests from osbuild-composer,
+specifically those CI is able to build.
+
+Importing manifests which cannot be built by CI will leave the
+DB in a dirty state. For this reason they are excluded.
+
+Once the import is done, a commit is created and pushed to [this GitLab repository](https://gitlab.com/redhat/services/products/image-builder/ci/manifest-db).
+
+#### Second step, running the CI
+
+Using a trigger token the github action the CI start. Ci will use an other
```

Comment:
1. First sentence has a syntax mistake.  Not sure what it wants to say exactly.
2. CI in the second sentence isn't properly captialised.